OSMP_FMU/esmini.h

```diff
@@ -196,22 +199,29 @@ class EsminiOsiSource {
     string fmuResourceLocation;
     bool visible;
     bool loggingOn;
+    bool slaveTerminated;

```

Comment:
Is "slaveTerminated" really the correct term here? Maybe something like "simulation_terminated" or "fmu_terminated" would be more accurate? It is not clear, what the "slave" is in this regard.